ООП Лабораторная работа 3

SurkovaSS opened the pull request.

SurkovaSS committed:
ООП Лабораторная работа 3

zagranet reviewed (changes requested):
- метод str перегружен в классах PaperBook, AudioBook - не нужно, т.к. вывод похож на тот, что в базовом классе
- метод repr не перегружен в классах PaperBook, AudioBook - нужно, т.к. кодовое представление у каждого класса свое
- остальное отлично

SurkovaSS committed:
Update task1.py

Перегружен метод __rerp__ из базового класса. Метод __str__ теперь наследуется.
